### ISSUE TITLE
"Unknown inResponseTo" when logging using SAML

### ISSUE THREAD

**Issue by vistaar-akhan** [mautic/mautic#13964]
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.2.21

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What happened?

- I installed mautic on Azure server.    
- I installed Certbot SSL on the domain
- I added authentication with google SAML and followed the Authentication documentation.

But when I use https://crm.my-domain.com/
It shows Unknown InResponseTo ‘_6b026a760b07afe943f44b87d32df905735c9774e3’

### Login Page screenshot
![image](https://github.com/user-attachments/assets/31baeb20-8acc-4167-a847-7b74bc4c93ac)

### Google SAML App configuration
![image](https://github.com/user-attachments/assets/597940f3-f1b4-4748-9daa-c3c84c774935)

### Mautic Configuration screenshot:
![image](https://github.com/user-attachments/assets/32692a8f-bc76-475b-a2bb-26dc1a9124fe)

**FYI: The normal username/password login is working fine. Issue is only with SSO**

### How can we reproduce this issue?

Step 1: Create a SAML app on google and fill all the information
Step 2: Download the IDP xml file 
Step 3: Configure Authentication on Mautic by uploading xml file of your idp which we downloaded in step 2

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
Thanks for the report, but this is a duplicate of an existing issue: https://github.com/mautic/mautic/issues/13808

Please continue the discussion there.

It's a tricky one as very few people use SAML who help with fixing and testing bugs, so if you can lend a hand with that it'd be most appreciated - see https://mau.tc/tester for docs on how to do that.
